In [1]:
import pandas as pd
import herramientas as h
import warnings

In [2]:
warnings.filterwarnings('ignore')

# Transformaciones

Importaremos la hoja 'HECHOS'

In [3]:
hechos = pd.read_excel('data/homicidios.xlsx', sheet_name='HECHOS')

In [4]:
hechos

ID  N_VICTIMAS      FECHA  AAAA  MM  DD      HORA  HH   
0    2016-0001           1 2016-01-01  2016   1   1  04:00:00   4  \
1    2016-0002           1 2016-01-02  2016   1   2  01:15:00   1   
2    2016-0003           1 2016-01-03  2016   1   3  07:00:00   7   
3    2016-0004           1 2016-01-10  2016   1  10  00:00:00   0   
4    2016-0005           1 2016-01-21  2016   1  21  05:20:00   5   
..         ...         ...        ...   ...  ..  ..       ...  ..   
691  2021-0093           1 2021-12-13  2021  12  13  17:10:00  17   
692  2021-0094           1 2021-12-20  2021  12  20  01:10:00   1   
693  2021-0095           1 2021-12-30  2021  12  30  00:43:00   0   
694  2021-0096           1 2021-12-15  2021  12  15  10:30:00  10   
695  2021-0097           1 2021-11-18  2021  11  18  06:10:00   6   

                               LUGAR_DEL_HECHO TIPO_DE_CALLE  ...  Altura   
0    AV PIEDRA BUENA Y AV FERNANDEZ DE LA CRUZ       AVENIDA  ...     NaN  \
1            AV GRAL PAZ  Y AV DE LOS CORRALES      GRAL PAZ  ...     NaN   
2                           AV ENTRE RIOS 2034       AVENIDA  ...  2034.0   
3        AV LARRAZABAL Y GRAL VILLEGAS CONRADO       AVENIDA  ...     NaN   
4    AV SAN JUAN Y PRESIDENTE LUIS SAENZ PEÑA        AVENIDA  ...     NaN   
..                                         ...           ...  ...     ...   
691                          AV. RIESTRA Y MOM       AVENIDA  ...     NaN   
692                    AU DELLEPIANE Y LACARRA     AUTOPISTA  ...     NaN   
693                        AV. GAONA Y TERRADA       AVENIDA  ...     NaN   
694                         AV. EVA PERON 4071       AVENIDA  ...  4071.0   
695                    PADRE CARLOS MUJICA 709         CALLE  ...   709.0   

                                   Cruce   
0    FERNANDEZ DE LA CRUZ, F., GRAL. AV.  \
1                    DE LOS CORRALES AV.   
2                                    NaN   
3               VILLEGAS, CONRADO, GRAL.   
4                SAENZ PE?A, LUIS, PRES.   
..                                   ...   
691                                  MOM   
692                          LACARRA AV.   
693                              TERRADA   
694                                  NaN   
695                                  NaN   

                                 Dirección Normalizada COMUNA   
0    PIEDRA BUENA AV. y FERNANDEZ DE LA CRUZ, F., G...      8  \
1                 PAZ, GRAL. AV. y DE LOS CORRALES AV.      9   
2                                  ENTRE RIOS AV. 2034      1   
3            LARRAZABAL AV. y VILLEGAS, CONRADO, GRAL.      8   
4              SAN JUAN AV. y SAENZ PEÑA, LUIS, PRES.      1   
..                                                 ...    ...   
691                                  RIESTRA AV. y MOM      7   
692         DELLEPIANE, LUIS, TTE. GRAL. y LACARRA AV.      9   
693                                GAONA AV. y TERRADA     11   
694                                PERON, EVA AV. 4071      9   
695                            PADRE CARLOS MUGICA 709      1   

                                   XY (CABA)         pos x         pos y   
0      Point (98896.78238426 93532.43437792)  -58.47533969  -34.68757022  \
1      Point (95832.05571093 95505.41641999)  -58.50877521  -34.66977709   
2     Point (106684.29090040 99706.57687843)  -58.39040293  -34.63189362   
3      Point (99840.65224780 94269.16534422)  -58.46503904  -34.68092974   
4    Point (106980.32827929 100752.16915795)  -58.38718297  -34.62246630   
..                                       ...           ...           ...   
691   Point (102728.60090138 98186.24929177)  -58.43353773  -34.64561636   
692    Point (99624.29795829 97569.69801131)  -58.46739825  -34.65117757   
693   Point (99116.45492358 101045.23284826)  -58.47293407  -34.61984745   
694    Point (99324.54463985 97676.26932409)  -58.47066794  -34.65021673   
695  Point (107664.16647795 104708.63962087)  -58.37976155  -34.58679619   

      PARTICIPANTES    VICTIMA    A

Reemplazamos los nombres de algunas columnas

In [5]:
new_names = {'LUGAR_DEL_HECHO': 'LUGAR', 'TIPO_DE_CALLE': 'TIPO_CALLE', 'Calle': 'CALLE', 'Altura': 'ALTURA', 'Cruce': 'CRUCE', 'Dirección Normalizada': 'DIRECCION', 'XY (CABA)': 'XY_(CABA)', 'pos x': 'POS_X', 'pos y': 'POS_Y'}

In [6]:
hechos = hechos.rename(columns=new_names)

In [7]:
hechos

ID  N_VICTIMAS      FECHA  AAAA  MM  DD      HORA  HH   
0    2016-0001           1 2016-01-01  2016   1   1  04:00:00   4  \
1    2016-0002           1 2016-01-02  2016   1   2  01:15:00   1   
2    2016-0003           1 2016-01-03  2016   1   3  07:00:00   7   
3    2016-0004           1 2016-01-10  2016   1  10  00:00:00   0   
4    2016-0005           1 2016-01-21  2016   1  21  05:20:00   5   
..         ...         ...        ...   ...  ..  ..       ...  ..   
691  2021-0093           1 2021-12-13  2021  12  13  17:10:00  17   
692  2021-0094           1 2021-12-20  2021  12  20  01:10:00   1   
693  2021-0095           1 2021-12-30  2021  12  30  00:43:00   0   
694  2021-0096           1 2021-12-15  2021  12  15  10:30:00  10   
695  2021-0097           1 2021-11-18  2021  11  18  06:10:00   6   

                                         LUGAR TIPO_CALLE  ...  ALTURA   
0    AV PIEDRA BUENA Y AV FERNANDEZ DE LA CRUZ    AVENIDA  ...     NaN  \
1            AV GRAL PAZ  Y AV DE LOS CORRALES   GRAL PAZ  ...     NaN   
2                           AV ENTRE RIOS 2034    AVENIDA  ...  2034.0   
3        AV LARRAZABAL Y GRAL VILLEGAS CONRADO    AVENIDA  ...     NaN   
4    AV SAN JUAN Y PRESIDENTE LUIS SAENZ PEÑA     AVENIDA  ...     NaN   
..                                         ...        ...  ...     ...   
691                          AV. RIESTRA Y MOM    AVENIDA  ...     NaN   
692                    AU DELLEPIANE Y LACARRA  AUTOPISTA  ...     NaN   
693                        AV. GAONA Y TERRADA    AVENIDA  ...     NaN   
694                         AV. EVA PERON 4071    AVENIDA  ...  4071.0   
695                    PADRE CARLOS MUJICA 709      CALLE  ...   709.0   

                                   CRUCE   
0    FERNANDEZ DE LA CRUZ, F., GRAL. AV.  \
1                    DE LOS CORRALES AV.   
2                                    NaN   
3               VILLEGAS, CONRADO, GRAL.   
4                SAENZ PE?A, LUIS, PRES.   
..                                   ...   
691                                  MOM   
692                          LACARRA AV.   
693                              TERRADA   
694                                  NaN   
695                                  NaN   

                                             DIRECCION COMUNA   
0    PIEDRA BUENA AV. y FERNANDEZ DE LA CRUZ, F., G...      8  \
1                 PAZ, GRAL. AV. y DE LOS CORRALES AV.      9   
2                                  ENTRE RIOS AV. 2034      1   
3            LARRAZABAL AV. y VILLEGAS, CONRADO, GRAL.      8   
4              SAN JUAN AV. y SAENZ PEÑA, LUIS, PRES.      1   
..                                                 ...    ...   
691                                  RIESTRA AV. y MOM      7   
692         DELLEPIANE, LUIS, TTE. GRAL. y LACARRA AV.      9   
693                                GAONA AV. y TERRADA     11   
694                                PERON, EVA AV. 4071      9   
695                            PADRE CARLOS MUGICA 709      1   

                                   XY_(CABA)         POS_X         POS_Y   
0      Point (98896.78238426 93532.43437792)  -58.47533969  -34.68757022  \
1      Point (95832.05571093 95505.41641999)  -58.50877521  -34.66977709   
2     Point (106684.29090040 99706.57687843)  -58.39040293  -34.63189362   
3      Point (99840.65224780 94269.16534422)  -58.46503904  -34.68092974   
4    Point (106980.32827929 100752.16915795)  -58.38718297  -34.62246630   
..                                       ...           ...           ...   
691   Point (102728.60090138 98186.24929177)  -58.43353773  -34.64561636   
692    Point (99624.29795829 97569.69801131)  -58.46739825  -34.65117757   
693   Point (99116.45492358 101045.23284826)  -58.47293407  -34.61984745   
694    Point (99324.54463985 97676.26932409)  -58.47066794  -34.65021673   
695  Point (107664.16647795 104708.63962087)  -58.37976155  -34.58679619   

      PARTICIPANTES    VICTIMA    ACUSADO  
0         MOTO-AUTO       M

Obtenemos la información del dataframe, como el tipo de datos, la cantidad de nulos y duplicados con una función del archivo herramientas.py

In [148]:
h.data_info(hechos)


Total rows:  696

Total full null rows:  0

Total duplicated rows: 0


Column                                              dType   
0              ID                                    [<class 'str'>]  \
1      N_VICTIMAS                                    [<class 'int'>]   
2           FECHA  [<class 'pandas._libs.tslibs.timestamps.Timest...   
3            AAAA                                    [<class 'int'>]   
4              MM                                    [<class 'int'>]   
5              DD                                    [<class 'int'>]   
6            HORA  [<class 'datetime.time'>, <class 'str'>, <clas...   
7              HH                     [<class 'int'>, <class 'str'>]   
8           LUGAR                                    [<class 'str'>]   
9      TIPO_CALLE                                    [<class 'str'>]   
10          CALLE                                    [<class 'str'>]   
11         ALTURA                                  [<class 'float'>]   
12          CRUCE                                    [<class 'str'>]   
13      DIRECCION                                    [<class 'str'>]   
14         COMUNA                                    [<class 'int'>]   
15      XY_(CABA)                                    [<class 'str'>]   
16          POS_X                                    [<class 'str'>]   
17          POS_Y                                    [<class 'str'>]   
18  PARTICIPANTES                                    [<class 'str'>]   
19        VICTIMA                                    [<class 'str'>]   
20        ACUSADO                                    [<class 'str'>]   

    No_Null_%  No_Null_Qty  Null_%  Null_Qty  
0      100.00          696    0.00         0  
1      100.00          696    0.00         0  
2      100.00          696    0.00         0  
3      100.00          696    0.00         0  
4      100.00          696    0.00         0  
5      100.00          696    0.00         0  
6      100.00          696    0.00         0  
7      100.00          696    0.00         0  
8      100.00          696    0.00         0  
9      100.00          696    0.00         0  
10      99.86          695    0.14         1  
11      18.53          129   81.47       567  
12      75.43          525   24.57       171  
13      98.85          688    1.15         8  
14     100.00          696    0.00         0  
15     100.00          696    0.00         0  
16     100.00          696    0.00         0  
17     100.00          696    0.00         0  
18     100.00          696    0.00         0  
19     100.00          696    0.00         0  
20     100.00          696    0.00         0

Corroboramos por qué en la columna 'HH' están presentes valores de tipo string y qué valores son

In [149]:
valores_string = hechos[hechos['HH'].apply(lambda x: isinstance(x, str))]['HH']

print(valores_string)

518    SD
Name: HH, dtype: object


Como podemos ver, en la fila 518 de la columna 'HH' hay un valor de tipo string y es 'SD', esto está representando que el valor es nulo. Por lo que también rellenaremos el resto de valores nulos con el valor 'SD'

In [150]:
hechos = hechos.fillna('SD')

In [151]:
h.data_info(hechos)


Total rows:  696

Total full null rows:  0

Total duplicated rows: 0


Column                                              dType   
0              ID                                    [<class 'str'>]  \
1      N_VICTIMAS                                    [<class 'int'>]   
2           FECHA  [<class 'pandas._libs.tslibs.timestamps.Timest...   
3            AAAA                                    [<class 'int'>]   
4              MM                                    [<class 'int'>]   
5              DD                                    [<class 'int'>]   
6            HORA  [<class 'datetime.time'>, <class 'str'>, <clas...   
7              HH                     [<class 'int'>, <class 'str'>]   
8           LUGAR                                    [<class 'str'>]   
9      TIPO_CALLE                                    [<class 'str'>]   
10          CALLE                                    [<class 'str'>]   
11         ALTURA                   [<class 'str'>, <class 'float'>]   
12          CRUCE                                    [<class 'str'>]   
13      DIRECCION                                    [<class 'str'>]   
14         COMUNA                                    [<class 'int'>]   
15      XY_(CABA)                                    [<class 'str'>]   
16          POS_X                                    [<class 'str'>]   
17          POS_Y                                    [<class 'str'>]   
18  PARTICIPANTES                                    [<class 'str'>]   
19        VICTIMA                                    [<class 'str'>]   
20        ACUSADO                                    [<class 'str'>]   

    No_Null_%  No_Null_Qty  Null_%  Null_Qty  
0       100.0          696     0.0         0  
1       100.0          696     0.0         0  
2       100.0          696     0.0         0  
3       100.0          696     0.0         0  
4       100.0          696     0.0         0  
5       100.0          696     0.0         0  
6       100.0          696     0.0         0  
7       100.0          696     0.0         0  
8       100.0          696     0.0         0  
9       100.0          696     0.0         0  
10      100.0          696     0.0         0  
11      100.0          696     0.0         0  
12      100.0          696     0.0         0  
13      100.0          696     0.0         0  
14      100.0          696     0.0         0  
15      100.0          696     0.0         0  
16      100.0          696     0.0         0  
17      100.0          696     0.0         0  
18      100.0          696     0.0         0  
19      100.0          696     0.0         0  
20      100.0          696     0.0         0

In [152]:
hechos

ID  N_VICTIMAS      FECHA  AAAA  MM  DD      HORA  HH   
0    2016-0001           1 2016-01-01  2016   1   1  04:00:00   4  \
1    2016-0002           1 2016-01-02  2016   1   2  01:15:00   1   
2    2016-0003           1 2016-01-03  2016   1   3  07:00:00   7   
3    2016-0004           1 2016-01-10  2016   1  10  00:00:00   0   
4    2016-0005           1 2016-01-21  2016   1  21  05:20:00   5   
..         ...         ...        ...   ...  ..  ..       ...  ..   
691  2021-0093           1 2021-12-13  2021  12  13  17:10:00  17   
692  2021-0094           1 2021-12-20  2021  12  20  01:10:00   1   
693  2021-0095           1 2021-12-30  2021  12  30  00:43:00   0   
694  2021-0096           1 2021-12-15  2021  12  15  10:30:00  10   
695  2021-0097           1 2021-11-18  2021  11  18  06:10:00   6   

                                         LUGAR TIPO_CALLE  ...  ALTURA   
0    AV PIEDRA BUENA Y AV FERNANDEZ DE LA CRUZ    AVENIDA  ...      SD  \
1            AV GRAL PAZ  Y AV DE LOS CORRALES   GRAL PAZ  ...      SD   
2                           AV ENTRE RIOS 2034    AVENIDA  ...  2034.0   
3        AV LARRAZABAL Y GRAL VILLEGAS CONRADO    AVENIDA  ...      SD   
4    AV SAN JUAN Y PRESIDENTE LUIS SAENZ PEÑA     AVENIDA  ...      SD   
..                                         ...        ...  ...     ...   
691                          AV. RIESTRA Y MOM    AVENIDA  ...      SD   
692                    AU DELLEPIANE Y LACARRA  AUTOPISTA  ...      SD   
693                        AV. GAONA Y TERRADA    AVENIDA  ...      SD   
694                         AV. EVA PERON 4071    AVENIDA  ...  4071.0   
695                    PADRE CARLOS MUJICA 709      CALLE  ...   709.0   

                                   CRUCE   
0    FERNANDEZ DE LA CRUZ, F., GRAL. AV.  \
1                    DE LOS CORRALES AV.   
2                                     SD   
3               VILLEGAS, CONRADO, GRAL.   
4                SAENZ PE?A, LUIS, PRES.   
..                                   ...   
691                                  MOM   
692                          LACARRA AV.   
693                              TERRADA   
694                                   SD   
695                                   SD   

                                             DIRECCION COMUNA   
0    PIEDRA BUENA AV. y FERNANDEZ DE LA CRUZ, F., G...      8  \
1                 PAZ, GRAL. AV. y DE LOS CORRALES AV.      9   
2                                  ENTRE RIOS AV. 2034      1   
3            LARRAZABAL AV. y VILLEGAS, CONRADO, GRAL.      8   
4              SAN JUAN AV. y SAENZ PEÑA, LUIS, PRES.      1   
..                                                 ...    ...   
691                                  RIESTRA AV. y MOM      7   
692         DELLEPIANE, LUIS, TTE. GRAL. y LACARRA AV.      9   
693                                GAONA AV. y TERRADA     11   
694                                PERON, EVA AV. 4071      9   
695                            PADRE CARLOS MUGICA 709      1   

                                   XY_(CABA)         POS_X         POS_Y   
0      Point (98896.78238426 93532.43437792)  -58.47533969  -34.68757022  \
1      Point (95832.05571093 95505.41641999)  -58.50877521  -34.66977709   
2     Point (106684.29090040 99706.57687843)  -58.39040293  -34.63189362   
3      Point (99840.65224780 94269.16534422)  -58.46503904  -34.68092974   
4    Point (106980.32827929 100752.16915795)  -58.38718297  -34.62246630   
..                                       ...           ...           ...   
691   Point (102728.60090138 98186.24929177)  -58.43353773  -34.64561636   
692    Point (99624.29795829 97569.69801131)  -58.46739825  -34.65117757   
693   Point (99116.45492358 101045.23284826)  -58.47293407  -34.61984745   
694    Point (99324.54463985 97676.26932409)  -58.47066794  -34.65021673   
695  Point (107664.16647795 104708.63962087)  -58.37976155  -34.58679619   

      PARTICIPANTES    VICTIMA    ACUSADO  
0         MOTO-AUTO       M

Ahora cambiaremos los valores flotantes a valores enteros en la columna 'ALTURA' y cambiaremos los valores en 'FECHA' y 'HORA' a tipo string

In [153]:
hechos['ALTURA'] = hechos['ALTURA'].apply(lambda x: int(x) if isinstance(x, (float, int)) else x)

In [154]:
hechos['FECHA'] = hechos['FECHA'].astype(str)

In [155]:
hechos['HORA'] = hechos['HORA'].astype(str)

In [156]:
h.data_info(hechos)


Total rows:  696

Total full null rows:  0

Total duplicated rows: 0


Column                           dType  No_Null_%  No_Null_Qty   
0              ID                 [<class 'str'>]      100.0          696  \
1      N_VICTIMAS                 [<class 'int'>]      100.0          696   
2           FECHA                 [<class 'str'>]      100.0          696   
3            AAAA                 [<class 'int'>]      100.0          696   
4              MM                 [<class 'int'>]      100.0          696   
5              DD                 [<class 'int'>]      100.0          696   
6            HORA                 [<class 'str'>]      100.0          696   
7              HH  [<class 'int'>, <class 'str'>]      100.0          696   
8           LUGAR                 [<class 'str'>]      100.0          696   
9      TIPO_CALLE                 [<class 'str'>]      100.0          696   
10          CALLE                 [<class 'str'>]      100.0          696   
11         ALTURA  [<class 'str'>, <class 'int'>]      100.0          696   
12          CRUCE                 [<class 'str'>]      100.0          696   
13      DIRECCION                 [<class 'str'>]      100.0          696   
14         COMUNA                 [<class 'int'>]      100.0          696   
15      XY_(CABA)                 [<class 'str'>]      100.0          696   
16          POS_X                 [<class 'str'>]      100.0          696   
17          POS_Y                 [<class 'str'>]      100.0          696   
18  PARTICIPANTES                 [<class 'str'>]      100.0          696   
19        VICTIMA                 [<class 'str'>]      100.0          696   
20        ACUSADO                 [<class 'str'>]      100.0          696   

    Null_%  Null_Qty  
0      0.0         0  
1      0.0         0  
2      0.0         0  
3      0.0         0  
4      0.0         0  
5      0.0         0  
6      0.0         0  
7      0.0         0  
8      0.0         0  
9      0.0         0  
10     0.0         0  
11     0.0         0  
12     0.0         0  
13     0.0         0  
14     0.0         0  
15     0.0         0  
16     0.0         0  
17     0.0         0  
18     0.0         0  
19     0.0         0  
20     0.0         0

In [157]:
hechos

ID  N_VICTIMAS       FECHA  AAAA  MM  DD      HORA  HH   
0    2016-0001           1  2016-01-01  2016   1   1  04:00:00   4  \
1    2016-0002           1  2016-01-02  2016   1   2  01:15:00   1   
2    2016-0003           1  2016-01-03  2016   1   3  07:00:00   7   
3    2016-0004           1  2016-01-10  2016   1  10  00:00:00   0   
4    2016-0005           1  2016-01-21  2016   1  21  05:20:00   5   
..         ...         ...         ...   ...  ..  ..       ...  ..   
691  2021-0093           1  2021-12-13  2021  12  13  17:10:00  17   
692  2021-0094           1  2021-12-20  2021  12  20  01:10:00   1   
693  2021-0095           1  2021-12-30  2021  12  30  00:43:00   0   
694  2021-0096           1  2021-12-15  2021  12  15  10:30:00  10   
695  2021-0097           1  2021-11-18  2021  11  18  06:10:00   6   

                                         LUGAR TIPO_CALLE  ... ALTURA   
0    AV PIEDRA BUENA Y AV FERNANDEZ DE LA CRUZ    AVENIDA  ...     SD  \
1            AV GRAL PAZ  Y AV DE LOS CORRALES   GRAL PAZ  ...     SD   
2                           AV ENTRE RIOS 2034    AVENIDA  ...   2034   
3        AV LARRAZABAL Y GRAL VILLEGAS CONRADO    AVENIDA  ...     SD   
4    AV SAN JUAN Y PRESIDENTE LUIS SAENZ PEÑA     AVENIDA  ...     SD   
..                                         ...        ...  ...    ...   
691                          AV. RIESTRA Y MOM    AVENIDA  ...     SD   
692                    AU DELLEPIANE Y LACARRA  AUTOPISTA  ...     SD   
693                        AV. GAONA Y TERRADA    AVENIDA  ...     SD   
694                         AV. EVA PERON 4071    AVENIDA  ...   4071   
695                    PADRE CARLOS MUJICA 709      CALLE  ...    709   

                                   CRUCE   
0    FERNANDEZ DE LA CRUZ, F., GRAL. AV.  \
1                    DE LOS CORRALES AV.   
2                                     SD   
3               VILLEGAS, CONRADO, GRAL.   
4                SAENZ PE?A, LUIS, PRES.   
..                                   ...   
691                                  MOM   
692                          LACARRA AV.   
693                              TERRADA   
694                                   SD   
695                                   SD   

                                             DIRECCION COMUNA   
0    PIEDRA BUENA AV. y FERNANDEZ DE LA CRUZ, F., G...      8  \
1                 PAZ, GRAL. AV. y DE LOS CORRALES AV.      9   
2                                  ENTRE RIOS AV. 2034      1   
3            LARRAZABAL AV. y VILLEGAS, CONRADO, GRAL.      8   
4              SAN JUAN AV. y SAENZ PEÑA, LUIS, PRES.      1   
..                                                 ...    ...   
691                                  RIESTRA AV. y MOM      7   
692         DELLEPIANE, LUIS, TTE. GRAL. y LACARRA AV.      9   
693                                GAONA AV. y TERRADA     11   
694                                PERON, EVA AV. 4071      9   
695                            PADRE CARLOS MUGICA 709      1   

                                   XY_(CABA)         POS_X         POS_Y   
0      Point (98896.78238426 93532.43437792)  -58.47533969  -34.68757022  \
1      Point (95832.05571093 95505.41641999)  -58.50877521  -34.66977709   
2     Point (106684.29090040 99706.57687843)  -58.39040293  -34.63189362   
3      Point (99840.65224780 94269.16534422)  -58.46503904  -34.68092974   
4    Point (106980.32827929 100752.16915795)  -58.38718297  -34.62246630   
..                                       ...           ...           ...   
691   Point (102728.60090138 98186.24929177)  -58.43353773  -34.64561636   
692    Point (99624.29795829 97569.69801131)  -58.46739825  -34.65117757   
693   Point (99116.45492358 101045.23284826)  -58.47293407  -34.61984745   
694    Point (99324.54463985 97676.26932409)  -58.47066794  -34.65021673   
695  Point (107664.16647795 104708.63962087)  -58.37976155  -34.58679619   

      PARTICIPANTES    VICTIMA    ACUSADO  
0         MOTO-AUTO       M

Finalmente guardamos los datos transformados en un archivo Excel

In [173]:
hechos.to_excel('data/hechos.xlsx', index=False)